ci.sh

```diff
@@ -50,6 +51,8 @@ case $HOST_TARGET in
     MIRI_TEST_TARGET=i686-unknown-linux-gnu run_tests
     MIRI_TEST_TARGET=aarch64-apple-darwin run_tests
     MIRI_TEST_TARGET=i686-pc-windows-msvc run_tests
+    # TODO: Enable once https://github.com/rust-lang/rust/pull/94813 lands
+    # MIRI_TEST_TARGET=aarch64-linux-android run_tests
```

Comment:
Could you add a note that @landaire is maintaining this target -- so that we know whom to ping when something android-specific breaks?